lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -789,6 +795,50 @@ def test_render_html_view_with_preview_mode_when_user_already_has_cert(self):
         self.assertIn('Signatory_Title 0', response.content)
 
     @override_settings(FEATURES=FEATURES_WITH_CERTS_ENABLED)
+    def test_html_view_certificate_availability_date_for_instructor_paced_courses(self):
```

Comment:
Oh, you'd also have to add some logic about what the expected date is before the assertion, like
```
if is_self_paced:
   expected_date = datetime.datetime.today()
else:
   expected_date = self.course.certificate_available_date